### ISSUE TITLE
SlowBuffer is undefined

### ISSUE THREAD

**Issue by december1981** [oven-sh/bun#1131]
### Version

0.1.10

### Platform

5.13.0-52-generic 20.04.1-Ubuntu SMP

### What steps will reproduce the bug?

The npm dependency causing the issue for me was

"jsonwebtoken": "^8.5.1"

And this happens when I import:

```js
import jwt from 'jsonwebtoken';
```

After upgrading from 0.1.8 to 0.1.10, this went from working fine to no longer working.

### How often does it reproduce? Is there a required condition?

This is 100%.

### What do you see instead?

call stack

```
36 | var origBufEqual = Buffer.prototype.equal;
37 | var origSlowBufEqual = SlowBuffer.prototype.equal;
                           ^
TypeError: undefined is not an object (evaluating 'SlowBuffer.prototype')
      at app/node_modules/buffer-equal-constant-time/index.js:37:23
      at bun:wrap:1:16354
      at app/node_modules/jwa/index.js:1:26
      at bun:wrap:1:16354
      at app/node_modules/jws/lib/sign-stream.js:4:18
      at bun:wrap:1:16354
      at app/node_modules/jws/index.js:2:25
      at bun:wrap:1:16354
      at app/node_modules/jsonwebtoken/decode.js:1:18
      at bun:wrap:1:16354
      at app/node_modules/jsonwebtoken/index.js:2:18
```

**Comment by Eckhardt-D** [comment]
Can confirm this on WSL Ubuntu-20.04 too.

**Comment by december1981** [comment]
I should comment in passing that while this import exception issue is now resolved (testing with 0.1.13), the jsonwebtoken functionality is nevertheless broken (it wasn't at 0.1.8). Signing to generate a token with bun works, but the jwt verify on this token is broken.

To confirm this, I signed using the module with bun to get a token, and confimed it correctly verifies with the same token and secret passed when verifying with node, but not bun.

index.js:

```js
import jwt from 'jsonwebtoken';

console.log(jwt.verify('<token generated from sign>', '<the secret used with sign>'));
```

```bash
> node index.js
{ user: 'test', iat: 1663665816, exp: 1680945816 }

> bun index.js
JsonWebTokenError: invalid token
```
